web/src/components/SpanDetail/components/GenericHttpSpanDetail.tsx

```diff
@@ -0,0 +1,53 @@
+import {FC, useState} from 'react';
+import CreateAssertionModal from 'components/CreateAssertionModal';
+import * as S from '../SpanDetail.styled';
+import TraceAnalyticsService from '../../../services/Analytics/TraceAnalytics.service';
+import {useAppSelector} from '../../../redux/hooks';
+import AssertionSelectors from '../../../selectors/Assertion.selectors';
+import {ISpanDetailProps} from '../SpanDetail';
+import {AssetionsSpanComponent} from './AssetionsSpanComponent';
+import {GenericHttpSpanHeader} from './GenericHttpSpanHeader';
+
+const {onAddAssertionButtonClick} = TraceAnalyticsService;
+
+const GenericHttpSpanDetail: FC<ISpanDetailProps> = ({testId, span, resultId}) => {
+  const [openCreateAssertion, setOpenCreateAssertion] = useState(false);
+
+  const assertionsResultList = useAppSelector(
+    AssertionSelectors.selectAssertionResultListBySpan(testId, resultId, span?.spanId)
+  );
+
+  return (
+    <>
+      <S.DetailsContainer>
+        <GenericHttpSpanHeader
+          title={`Details for ${span?.type?.toUpperCase()} ${span?.name}`}
```

Comment:
I think we should move this to the `SpanDetail` facade component as we want to show this for every span.